spec/features/5_photos_spec.rb

```diff
@@ -1,120 +1,126 @@
-    user_3 = FactoryGirl.create(:user, :username => "Three", :email => "three@m.com")
-    photo_1 = FactoryGirl.create(:photo, :user_id => user_1.id)
-    photo_2 = FactoryGirl.create(:photo, :user_id => user_2.id)
-    login_as(user_3, :scope => :user)
+      visit "/photos"
 
-    visit "/photos"
+      expect(page).to have_css(".fa-plus")
+      expect(page).not_to have_selector("h1", text: "Photos")
+    end
 
-    expect(page).to have_content("#{photo_1.user.username}")
-    expect(page).to have_css("img[src*='#{photo_1.image}']")
-    expect(page).to have_content("#{photo_2.user.username}")
-    expect(page).to have_css("img[src*='#{photo_2.image}']")
-    expect(page).to have_content("ago")
-  end
+    scenario "/photos displays per-photo username, photo, and time elapsed", points: 1, hint: h("time_in_words") do
+      login_as(@user_3, :scope => :user)
 
-  scenario "header uses Font Awesome fa-sign-out icon", points: 1 do
-    user = FactoryGirl.create(:user)
-    login_as(user, :scope => :user)
+      visit "/photos"
+      photos = Photo.all
 
-    visit "/photos"
+      photos.each do |photo|
+        expect(page).to have_content("#{photo.user.username}")
+        expect(page).to have_css("img[src*='#{photo.image}']")
+        expect(page).to have_content(ApplicationController.helpers.time_ago_in_words(photo.created_at))
+      end
+    end
 
-    within("nav") {
-      expect(page).to have_css(".fa-sign-out")
-    }
-  end
+    scenario "header uses Font Awesome fa-sign-out icon", points: 1 do
+      user = FactoryGirl.create(:user)
+      login_as(user, :scope => :user)
 
-  scenario "/photos lists comments with authors", points: 1 do
-    user_1 = FactoryGirl.create(:user, :username => "1", :email => "1@m.com")
-    user_2 = FactoryGirl.create(:user, :username => "2", :email => "2@m.com")
-    user_3 = FactoryGirl.create(:user, :username => "Three", :email => "three@m.com")
-    photo_1 = FactoryGirl.create(:photo, :user_id => user_3.id)
-    photo_2 = FactoryGirl.create(:photo, :user_id => user_3.id)
-    photo_3 = FactoryGirl.create(:photo, :user_id => user_3.id)
-    comment_1 = FactoryGirl.create(:comment, :user_id => user_1.id, :body => "comment_1", :photo_id => photo_1.id)
-    comment_2 = FactoryGirl.create(:comment, :user_id => user_2.id, :body => "comment_two", :photo_id => photo_3.id)
-    login_as(user_3, :scope => :user)
+      visit "/photos"
 
-    visit "/photos"
+      within("nav") {
+        expect(page).to have_css(".fa-sign-out")
+      }
+    end
 
-    expect(page).to have_content(comment_1.body)
-    expect(page).to have_content(comment_1.user.username)
-    expect(page).to have_content(comment_2.body)
-    expect(page).to have_content(comment_2.user.username)
-  end
+    scenario "/photos lists comments with authors", points: 1 do
+      comment_1 = FactoryGirl.create(:comment, :user_id => @user_1.id, :body => "comment_1", :photo_id => @photo_1.id)
```

Comment:
Notice how far away this test is from the creation of `@user_1` — this is why using `before` and `let` leads to tests that are difficult to read in isolation.

In any case, we should be able to do something like `photos = create_list(:photo_with_comments, 3)` using FactoryBot fatures to do most of this work more legibly?